#10516 (#10741)
MicrosoftDocs/windows-itpro-docs: windows/security/threat-protection/windows-sandbox/windows-sandbox-overview.md

4. Locate and select **Windows Sandbox** on the Start menu to run it for the first time.

   > [!NOTE]

>    > The Sandbox version of Windows will not respect the mouse settings of the host system, so if the host system is set to use a right-handed mouse, you will need to apply these settings inside the Sandbox manually.  

Reviewer: ```suggestion
   > Windows Sandbox does not adhere to the mouse settings of the host system, so if the host system is set to use a right-handed mouse, you should apply these settings in Windows Sandbox manually.  
```
